exercises/enmanuel-delanuez/answers.js

```diff
@@ -0,0 +1,109 @@
+// Question 1
+function greet(name) {
+  return (`Good morning, ${name}!`);
+}
+
+// Question 2 --change 99 to a parameter for expandability
+function logOddNumbers() {
+  for (let i = 1; i <= 99; i++) {
+    if (i % 2 === 1) {
+      console.log(i);
+    }
+  }
+}
+
+//let area = (w, h) => w * h;
+// Question 3
+function area(w, h) {
+  return w * h;
+}
+
+//let otherAngle = (angOne, angTwo) => (180 - (angOne + angTwo));
+// Question 4
+function otherAngle(angOne, angTwo) {
+  return (180 - (angOne + angTwo));
+}
+
+//let diameterFromArea = (area) => 2 * (Math.sqrt(area / Math.PI));
+// Question 5
+function diameterFromArea(area) {
+  return 2 * (Math.sqrt(area / Math.PI));
+}
+
+
+//let totalPrice = (gross, tax) => gross + (gross * (tax / 100));
+// Question 6
+function totalPrice(gross, tax) {
+  return gross + (gross * (tax / 100));
+}
+
+// Question 7
+function shortLongShort(strngOne, strngTwo) {
+  let final = ''
+  if (strngOne.length < strngTwo.length) {
+    final = strngOne + strngTwo + strngOne;
+  } else {
+    final = strngTwo + strngOne + strngTwo;
+  }
+  return final;
+}
+
+// Question 8
+function describeAge(age) {
+  return age < 12 ? "You're a kid.": age > 11 && age <18 ? "You're a teenager.": age >= 18 && age <= 64 ? "You're an adult.": "You're an elderly.";
+}
+
+// // Question 9
+function isLeapYear(year) {
+  return ((year % 4 === 0) && (year % 100 != 0)) || (year % 400 === 0) ? true : false
```

Comment:
No need for the ternary statement right.... what happens if we just return `((year % 4 === 0) && (year % 100 != 0)) || (year % 400 === 0)`